src/pages/SingleTweetComment.tsx

```diff
@@ -0,0 +1,243 @@
+import { Dropdown } from "react-bootstrap";
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  max-width: 598px;
+  text-align: left;
+  display: flex;
+  flex-direction: row;
+  padding: 0 16px;
+  border-top: solid 1.5px #e6e6e6;
+  border-bottom: solid 1.5px #e6e6e6;
+  &:hover {
+    background-color: rgba(0, 0, 0, 0.03);
+  }
+`;
+const CommentWrapper = styled.div`
+  width: 100%;
+`;
+const Logo = styled.img`
+  width: 48px;
+  height: 48px;
+  margin: 12px 12px 0 0;
+  border-radius: 50%;
+`;
+const Author = styled.div`
+  margin-top: 12px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  letter-spacing: -0.02em;
+`;
+const NameSection = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+`;
+const NameWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+const Name = styled.div`
+  font-weight: bold;
+  font-size: 16px;
+  height: 20px;
+  display: flex;
+  align-items: center;
+`;
+const NickName = styled.div`
+  font-size: 16px;
+  color: rgb(83, 100, 113);
+  height: 20px;
+  display: flex;
+  align-items: center;
+  margin-left: 4px;
+`;
+const Date = styled.div`
+  color: rgb(83, 100, 113);
+  height: 20px;
+  display: flex;
+  align-items: center;
+  margin-left: 4px;
+`;
+const SDropdownToggle = styled(Dropdown.Toggle)`
+  border-radius: 50%;
+  background-color: white;
+  width: 18.75px;
+  height: 18.75px;
+  padding: 0;
+  border: none;
+  margin-left: 20px;
+  &:hover {
+    background-color: white;
+  }
+`;
+const Answer = styled.div`
+  width: 100%;
+  display: flex;
+`;
+const Answertext = styled.div`
+  font-size: 16px;
+  color: rgb(83, 100, 113);
+  height: 20px;
+  display: flex;
+  align-items: center;
+`;
+const Text = styled.div`
+  font-size: 1rem;
+  letter-spacing: -0.02em;
+  line-height: 1.6rem;
+`;
+
+const IconRow = styled.div`
+  max-width: 425px;
+  height: 20px;
+  margin: 12px 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`;
+const IconBox = styled.div`
+  display: flex;
+`;
+
+interface IIconBg {
+  iconBgColor: string;
+}
+const IconBg = styled.div<IIconBg>`
+  height: 34.75px;
+  width: 34.75px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50%;
+  &:hover {
+    background-color: ${(props) => props.iconBgColor};
+  }
+`;
+
+interface IIcon {
```

Comment:
```suggestion
interface IIcon {
```
you declarate such interface in two different module. Create it in one place and import it in seconde

Reply:
This is Artem's component. And it is already in develop branch